browser/src/control/jsdialog/Widget.TreeView.js

```diff
@@ -80,23 +80,27 @@ function _createCheckbox(parentContainer, treeViewData, builder, entry) {
 
 	if (treeViewData.enabled !== false && treeViewData.enabled !== 'false') {
 		$(checkbox).change(function() {
-			if (this.checked) {
-				var foundEntry = _findEntryWithRow(treeViewData.entries, entry.row);
-				if (foundEntry)
-					foundEntry.state = true;
-				builder.callback('treeview', 'change', treeViewData, {row: entry.row, value: true}, builder);
-			} else {
-				var foundEntry = _findEntryWithRow(treeViewData.entries, entry.row);
-				if (foundEntry)
-					foundEntry.state = false;
-				builder.callback('treeview', 'change', treeViewData, {row: entry.row, value: false}, builder);
-			}
+			_chnageCheckboxStateOnClick(this, treeViewData, builder, entry);
 		});
 	}
 
 	return checkbox;
 }
 
+function _chnageCheckboxStateOnClick(checkbox, treeViewData, builder, entry) {
```

Comment:
shouldn't this be `_changeCheckboxStateOnClick` (see typo) ?

Reply:
i see thanks for the save 🙇🏼